[2022-08-11 23:44:54,735] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ETL_Raízen.download_raw_data manual__2022-08-11T23:44:45.811252+00:00 [queued]>
[2022-08-11 23:44:54,773] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ETL_Raízen.download_raw_data manual__2022-08-11T23:44:45.811252+00:00 [queued]>
[2022-08-11 23:44:54,775] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-08-11 23:44:54,776] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-08-11 23:44:54,777] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-08-11 23:44:54,831] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): download_raw_data> on 2022-08-11 23:44:45.811252+00:00
[2022-08-11 23:44:54,848] {standard_task_runner.py:52} INFO - Started process 6860 to run task
[2022-08-11 23:44:54,858] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ETL_Raízen', 'download_raw_data', 'manual__2022-08-11T23:44:45.811252+00:00', '--job-id', '96', '--raw', '--subdir', 'DAGS_FOLDER/ETL_Raizen.py', '--cfg-path', '/tmp/tmpgyo29jhh', '--error-file', '/tmp/tmpk8uczeky']
[2022-08-11 23:44:54,860] {standard_task_runner.py:80} INFO - Job 96: Subtask download_raw_data
[2022-08-11 23:44:55,043] {task_command.py:369} INFO - Running <TaskInstance: ETL_Raízen.download_raw_data manual__2022-08-11T23:44:45.811252+00:00 [running]> on host 53e7d4a502e6
[2022-08-11 23:44:55,713] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ETL_Raízen
AIRFLOW_CTX_TASK_ID=download_raw_data
AIRFLOW_CTX_EXECUTION_DATE=2022-08-11T23:44:45.811252+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-11T23:44:45.811252+00:00
[2022-08-11 23:44:55,726] {ETL_Raizen.py:41} INFO - Downloading .xls raw data
[2022-08-11 23:44:59,610] {ETL_Raizen.py:43} INFO - Succesfully downloaded .xls raw data
[2022-08-11 23:44:59,615] {python.py:173} INFO - Done. Returned value was: None
[2022-08-11 23:44:59,667] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=ETL_Raízen, task_id=download_raw_data, execution_date=20220811T234445, start_date=20220811T234454, end_date=20220811T234459
[2022-08-11 23:44:59,784] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-11 23:45:00,143] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
